.github/workflows/master.yml

```diff
@@ -176,6 +176,10 @@ jobs:
         export PATH="$CONDA_DIR/bin:$PATH"
         source activate test-environment
         ./dev/run-large-python-tests.sh
+    - name: Clean-up test-environment
+      run: |
+        conda remove --all --yes --name test-environment
+        conda clean --all --yes
```

Comment:
Fix for this error in `test-anaconda-compatibility.sh`:

```
docker: failed to register layer: Error processing tar file(exit status 1): write /opt/conda/pkgs/mkl-2019.3-199.tar.bz2: no space left on device.
```

https://github.com/mlflow/mlflow/runs/2163733721#step:6:132